tests/range/CMakeLists.txt

```diff
@@ -26,11 +26,5 @@ Range of dims 3
 0
 1
 0
-1
-0
-0
-1
-1
-1
```

Comment:
Is this an issue for the CTS ?
It is a unit test for triSYCL, not for ComputeCpp or the CTS...
Similar to the discussion in https://github.com/triSYCL/triSYCL/issues/174

Reply:
These are removed because the spec doesn't contain them, and adding bool casting operators is on the road to ambiguity.